### ISSUE TITLE
Can't activate all auf the selected clima options over the preheating service

### ISSUE THREAD

**Issue by Keckeldeckel** [Hyundai-Kia-Connect/kia_uvo#827]
Please check Services, Known Bug / Issues and Troubleshooting over here first: https://github.com/fuatakgun/kia_uvo/blob/master/README.md
**Region and Brand of car**
 Hyundai Kona 2021 150kw / Europe / Germany
**Version of the integration**
2.20.2

**Describe the bug**
When I call this service like this
![2024-02-13 08_03_55-Entwicklerwerkzeuge – Home Assistant](https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/82964460/1be0835f-c3e2-42e6-a799-c1be73d72793)

The car won't heat the seats and the stearing weel. Just activate AC and defroster

**Comment by cdnninja** [comment]
Does your car have these options in the native app?

**Comment by Keckeldeckel** [comment]
Yes it has, you can set in the app witch parts will be used for preheating.

**Comment by cdnninja** [comment]
Have you tried other options.  For seats I would expect your car in the native app to offer levels.  Not "on" "off". If so select levels and see if that works.  

For steering. Try the other radios.  We may have the wrong mapping.

**Comment by Keckeldeckel** [comment]
@cdnninja Yes I have but it hasn't affected the problem. Also the steering wheel isn't heating up
